Packs/ExpanseV2/Scripts/ExpanseGenerateIssueMapWidgetScript/ExpanseGenerateIssueMapWidgetScript.py

```diff
@@ -21096,7 +21097,11 @@ def generate_map_command(args: Dict[str, Any]) -> str:
 
     annotations_image = Image.new('RGBA', (RESULT_IMAGE_X * 2, RESULT_IMAGE_Y * 2), (255, 0, 0, 0))
     draw = ImageDraw.Draw(annotations_image)
-    font = ImageFont.truetype("/usr/share/fonts/truetype/dejavu/DejaVuSansMono-Bold.ttf", 20)
+    if Path("/usr/share/fonts/truetype/dejavu/DejaVuSansMono-Bold.ttf"):  # debian location
+        deja_sans_mono_font_path = "/usr/share/fonts/truetype/dejavu/DejaVuSansMono-Bold.ttf"
+    else:  # ubi location
+        deja_sans_mono_font_path = "/usr/share/fonts/dejavu-sans-mono-fonts/DejaVuSansMono-Bold.ttf"
```

Comment:
this needs to be tested on an actual xsoarng machine, since we saw Sagi copies the folders to be sub some other folder in the runner

Reply:
no problem, you want it to be tested after Sagy builds up the engine image, right?

Reply:
yes, so lets not merge the change until then

Reply:
removed all of this its working without the change.